scripts/Package.swift

```diff
@@ -51,8 +51,8 @@ let package = Package(
         ),
         .binaryTarget(
             name: "swiftformat",
-            url: "https://github.com/nicklockwood/SwiftFormat/releases/download/0.54.0/swiftformat.artifactbundle.zip",
-            checksum: "edf4ed2f1664ad621ae71031ff915e0c6ef80ad66e87ea0e5a10c3580a27a6dd"
+            url: "https://github.com/nicklockwood/SwiftFormat/releases/download/0.54.6/swiftformat.artifactbundle.zip",
```

Comment:
Could you please also update `.github/workflows/swiftformat.yml` if there is an image for that version available?

Reply:
I've changed this to `0.54.5` as this is the latest image available and updated `.github/workflows/swiftformat.yml`. Thanks for pointing that out!